### PR TITLE
Update aks vscode extension document.

### PR DESCRIPTION
**Pull request by Tatsinnit**
This PR updates the new changes and some improvements around the `aks-extension`.

The new images etc, can be seen in my fork as fyi: https://github.com/Tatsinnit/vscode-docs/blob/update/aks-extension-docs/docs/azure/aksextensions.md 

Thanks heaps in advance for review. FYI : @gregvanl ,  @peterbom and @rzhang628 ☕️💡

### CODE REVIEW

**Review by peterbom — APPROVED** [review]

File: docs/azure/aksextensions.md [hunk 1]
```diff
@@ -141,6 +145,17 @@ Right-click on your AKS cluster and select **Show Properties** to display the AK
 
 ![Start or Stop Cluster from properties webview](images/aksextensions/aks-startstop-cluster.png)
 
+### Run Kubectl Commands from your AKS cluster
+
+Right click on your AKS cluster and click on **Run Kubectl Commands** to easily run few known kubectl commands on your cluster. Currently we have enable following kubectl commands for the AKS cluster:
```
Comment (gregvanl) [inline docs/azure/aksextensions.md:150]: "Right-click on your AKS cluster and select **Run KubectlCommands** to run Kubectl commands on your cluster. Currently, the following Kubectl commands are available:"

**Comment by gregvanl** [comment]
Hi @Tatsinnit, Thank you for updating this topic!
To add or update images in the vscode-docs repo, you need to have Git LFS installed and enabled. 
https://code.visualstudio.com/docs/languages/markdown#_markdown-preview has guidance for setting up Git LFS.
Using Git LFS, lets us keep large binary files like images and .gifs out of the repo to make cloning faster.

**Comment by gregvanl** [comment]
Can you also update the DateApproved metadata in the header to let users know the topic has been reviewed and updated?
We show the DateApproved at the bottom of each page and lets us determine topic freshness.

**Comment by gregvanl** [comment]
Hi @Tatsinnit The images still aren't in Git LFS so I created a new matching PR #5817 to get this merged.
You may want to reclone your vscode-docs fork and see if that helps you get Git LFS correctly set up.

**Pull request closed without merging**